BraveWallet/Crypto/Stores/KeyringStore.swift

```diff
@@ -366,3 +360,11 @@ extension KeyringStore: BraveWalletKeyringControllerObserver {
     updateKeyringInfo()
   }  
 }
+
+extension KeyringStore: SubStore {
+  public func resetStore() {
+    controller.reset()
+    isOnboardingVisible = true
+    Self.resetKeychainStoredPassword()
```

Comment:
This behaviour doesn't make sense in `resetStore`, in all other stores you're resetting local state, in this store you are resetting the wallet and removing biometrics, which is part of resetting the wallet but not part of resetting the store